app/subscriber/src/features/my-reports/edit/ReportEditMenu.tsx

```diff
@@ -35,7 +36,7 @@ export const ReportEditMenu = ({ onChange }: IReportEditMenuProps) => {
           <MenuButton
             label="Settings"
             active={!active || active?.startsWith(ReportMainMenuOption.Settings)}
-            onClick={() => onChange?.(`/reports/${values.id}/settings`)}
+            onClick={() => onChange?.(`/reports/${values.id}/${ReportMainMenuOption.Settings}`)}
```

Comment:
Improving maintainability of these links.  They are all managed by the enum values now.